src/features/pagination/templates/pagination.html

```diff
@@ -1,17 +1,14 @@
 <div
-  role="contentinfo"
   class="ui-grid-pager-panel"
   ui-grid-pager
   ng-show="grid.options.enablePaginationControls">
   <div
     role="navigation"
     class="ui-grid-pager-container">
     <div
-      role="menubar"
```

Comment:
Isn't the bottom bar a `menubar`? This is more for my own learning more than anything else.